java/src/jmri/jmrit/symbolicprog/SplitEnumVariableValue.java

```diff
@@ -511,7 +507,7 @@ void exitField(){
         if (_value != null && !oldContents.equals(_value.getSelectedItem())) {
             long newFieldVal = 0;
             try {
-                newFieldVal = Long.parseLong((String)_value.getSelectedItem());
+                newFieldVal = Long.parseLong((String) Objects.requireNonNull(_value.getSelectedItem()));
             } catch (NumberFormatException e) {
```

Comment:
What's the intent here? That the method throws a NPE, or that the catch statement catches the NPE? If the latter case, the catch statement should include NPE as well.

Reply:
This fix was suggested by the IDE. It should handle the case of nothing in the combo being selected. Explicit catch seems to reach the same goal.